pbl

```diff
@@ -265,6 +286,11 @@ if($program eq 'pbl') {
     'help'         => sub { pbl_usage 0 },
   ) || pbl_usage 1;
 }
+else {
+  GetOptions(
+    'help'         => sub { update_bootloader_usage 0 },
```

Comment:
maybe handle here that reinit instead of at https://github.com/openSUSE/perl-bootloader/pull/124/files#diff-25f66d09583995c8fbf858e439e0ab33R312 ?

Reply:
Actually, it is not a 'maybe' - otherwise the option will be reported as an invalid one :-)